src/uri/validator/urivalidator.rs

```diff
@@ -163,9 +163,9 @@ impl UriValidator {
                     .as_ref()
                     .map_or(false, |instance| instance.contains("response"));
```

Comment:
Also noticed that we're not checking for `name` to be "rpc", unlike within `is_rpc_method()`, even though the spec says to check the name is "rpc" for both validators.

Am I reading that right? Can you check me here again @AnotherDaniel?

Reply:
Maybe I'm wrong, but I guess it's intended to check "rpc" by using `is_rpc_method` [here](https://github.com/eclipse-uprotocol/up-rust/pull/52/files#diff-50d5281e2de4d6b2de3ffb2294e4a0bb1fe84a474bd1595b080e740cba9e2aacR159).

Reply:
[Spec says](https://github.com/eclipse-uprotocol/up-spec/blob/main/basics/uri.adoc#3-validators) we should check for it:
![image](https://github.com/eclipse-uprotocol/up-rust/assets/11622119/a6b3cca6-ba7f-4bfd-97db-9d3e8768d603)

Reply:
Updated.